tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -37,6 +37,10 @@ build --action_env=GCC_HOST_COMPILER_PATH="/dt7/usr/bin/gcc"
 build --action_env=LD_LIBRARY_PATH="/usr/local/cuda:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/tensorrt/lib"
 build --crosstool_top=@ubuntu18.04-gcc7_manylinux2010-cuda11.2-cudnn8.1-tensorrt7.2_config_cuda//crosstool:toolchain
 
+# Set up a manylinux2014 config option
```

Comment:
I'd also suggest that you drop the option here and just make manylinux2014 the default while you're using this PR. It will be easier to test the WIP containers that way, because they'll be the exact same as the regular ones and you won't need to change any flags outside.

We can revisit the preferred method of integration once you've verified everything works and you're ready to merge these -- which I expect won't happen for a while, until you've done more testing.

Reply:
Oh sure, I'll push a commit to update this. Thanks!